python/pdstools/adm/ADMDatamart.py

```diff
@@ -1916,10 +1944,9 @@ def _run_quarto_command(
                 check=True,
             )
             quarto_version = version_result.stdout.strip()
-            logging.info(f"Quarto version: {quarto_version}")
-            # add version compatibility check here if needed
+            log_and_write(f"Quarto version: {quarto_version}")
         except subprocess.CalledProcessError as e:
-            logging.warning(f"Failed to check Quarto version: {e}")
+            log_and_write(f"WARNING: Failed to check Quarto version: {e}")
```

Comment:
Just one of those examples where I think OOTB logging makes more sense: `logging.warning()` makes your implicit warning here, explicit.